### PR TITLE
Fix: "Commands"

### PR DESCRIPTION
**Pull request by DenisBratuh**
Description: refactor ONLY Commands(code will not compile because enums with result of operations was deleted) If the following code style is ok, then we will refactor next classes according to this one

### CODE REVIEW

File: src/main/java/com/andersentask/bookshop/console/Commands.java [hunk 1]
```diff
@@ -91,73 +69,49 @@ public List<Book> getSortedBooks(BookSort bookSort) {
...
      */
-    public ResultOfOperation.CreateOrder createOrder(List<Long> ids) {
-        List<Book> booksToOrder = appContextConfig.getBookService().getBooksByIds(ids);
-
-        ResultOfOperation.CreateOrder resultOfMethod = ResultOfOperation.CreateOrder.WRONG_BOOK_ID;
+    public void createOrder(List<Long> bookIds) {
+        List<Book> booksToOrder = appContextConfig.getBookService().getBooksByIds(bookIds);
 
-        if (booksToOrder.size() == ids.size()) {
+        if (booksToOrder.size() != 0) {
```
Comment (ulad-sachkovski) [inline src/main/java/com/andersentask/bookshop/console/Commands.java:79]: Why have u changed the logic?
We create order, only if all books are entered correctly
Now the order is created, even if user enters some non-existing book ids

File: src/main/java/com/andersentask/bookshop/console/Commands.java [hunk 2]
```diff
@@ -91,73 +69,49 @@ public List<Book> getSortedBooks(BookSort bookSort) {
...
-                    appContextConfig.getOrderService().changeStatusOfOrder(id, orderStatus);
-                }
-            } else appContextConfig.getOrderService().changeStatusOfOrder(id, orderStatus);
-
-            if (previousStatus.equals(order.getOrderStatus())) {
-                resultOfMethod = ResultOfOperation.ChangeStatusOfOrderIncludingBooksCheck.ORDER_STATUS_CAN_NOT_BE_UPDATED;
-            } else resultOfMethod = ResultOfOperation.ChangeStatusOfOrderIncludingBooksCheck.STATUS_UPDATED;
-        }
-        return resultOfMethod;
+    public void changeStatusOfOrderIncludingBooksCheck(Long id, OrderStatus orderStatus) {
```
Comment (ulad-sachkovski) [inline src/main/java/com/andersentask/bookshop/console/Commands.java:109]: if the status of the order was not changed, we should throw an exception
we somehow should trace, if the status is really changed

File: src/main/java/com/andersentask/bookshop/console/Commands.java [hunk 3]
```diff
@@ -91,73 +69,49 @@ public List<Book> getSortedBooks(BookSort bookSort) {
...
+        appContextConfig.getOrderService()
+                .getOrderById(id)
+                .filter(order -> !order.getOrderStatus().equals(orderStatus))
```
Comment (ulad-sachkovski) [inline src/main/java/com/andersentask/bookshop/console/Commands.java:112]: order.getOrderStatus() != orderStatus

File: src/main/java/com/andersentask/bookshop/console/Commands.java [hunk 4]
```diff
@@ -25,58 +24,37 @@ public class Commands {
...
-        Optional<Book> optionalBook = appContextConfig.getBookService().getBookById(id);
-        ResultOfOperation.SetBookStatus resultOfMethod = ResultOfOperation.SetBookStatus.WRONG_BOOK_ID;
+    public void setStatusToBookAndDeleteCorrespondingRequests(Long id, BookStatus bookStatus) {
 
-        if (optionalBook.isPresent()) {
-            Book book = optionalBook.get();
-            resultOfMethod = ResultOfOperation.SetBookStatus.BOOK_ALREADY_HAS_THIS_STATUS;
+        Book book = appContextConfig.getBookService()
+                .getBookById(id)
+                .orElseThrow(() -> new RuntimeException("Entity not found"));
```
Comment (ulad-sachkovski) [inline src/main/java/com/andersentask/bookshop/console/Commands.java:37]: Entity => book

File: src/main/java/com/andersentask/bookshop/console/Commands.java [hunk 5]
```diff
@@ -25,58 +24,37 @@ public class Commands {
...
-            if (!book.getStatus().equals(bookStatus)) {
-
-                if (book.getStatus().equals(BookStatus.OUT_OF_STOCK) && bookStatus.equals(BookStatus.AVAILABLE)) {
-                    appContextConfig.getRequestService().deleteRequest(book);
-                }
-
-                appContextConfig.getBookService().setStatusToBook(id, bookStatus);
-                resultOfMethod = ResultOfOperation.SetBookStatus.BOOK_STATUS_UPDATED;
-            }
+        if (book.getStatus().equals(BookStatus.OUT_OF_STOCK) && bookStatus.equals(BookStatus.AVAILABLE)) {
```
Comment (ulad-sachkovski) [inline src/main/java/com/andersentask/bookshop/console/Commands.java:39]: .equals => ==

File: src/main/java/com/andersentask/bookshop/console/Commands.java (hunk 1 above)
Comment (DenisBratuh) [reply src/main/java/com/andersentask/bookshop/console/Commands.java:79]: What is the purpose of this check?

File: src/main/java/com/andersentask/bookshop/console/Commands.java (hunk 2 above)
Comment (DenisBratuh) [reply src/main/java/com/andersentask/bookshop/console/Commands.java:109]: to discuss

**Comment by skfl** [comment]
Closed due duplicate of PR https://github.com/skfl/andersen-task/pull/65

**Pull request closed without merging**